### ISSUE TITLE
Make AsciiBib examples realistic and complete

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton.com#11]
Right now the examples used are half-true and half-made-up. We need to migrate to realistic examples with no filler information. I suggest we use ISO 19115-3 as a document example (since the content was already there).

The relevant files are:
* `_specs/asciibib-path.adoc`
* `_specs/asciibib-nested.adoc`

@manuel489 @anermina do you have time for this? Thanks!

**Comment by manuel489** [comment]
@ronaldtse , do you mean, for example, to pass from this:
````
[source,asciidoc]
----
[[iso123]]
[%bibitem]
== Rubber latex -- Sampling
docid::
  type::: ISO
  id::: ISO 123
type:: standard
----
````

to something like this (?): 

````
[source,asciidoc]
----
[[iso19115-3]]
[%bibitem]
== {blank}
docid::
  type::: ISO
  id::: 19115-3
type:: standard
----
````

**Comment by ronaldtse** [comment]
Yes indeed! More realistic examples so people can really understand without getting confused. Fake examples are often hard to follow.

**Comment by manuel489** [comment]
Hi @ronaldtse , apparently, I don't have enough privileges to push in this repository. Could you please take action on it?

**Comment by ronaldtse** [comment]
@manuel489 you have access now. Can you push to a new branch and make a PR? Thanks!

**Comment by ronaldtse** [comment]
Closing — this was addressed during the VitePress migration. The current specs/asciibib.md already uses realistic ISO 19115-3 examples (as requested), documents both nested and path-style syntaxes comprehensively, and includes complete examples for both approaches.
